.github/workflows/ci-on-pr-review.yaml

```diff
@@ -0,0 +1,37 @@
+name: Kata Containers CI
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  metadata:
+    runs-on: ubuntu-latest
+    outputs:
+      has_test_command: ${{ steps.test-command.outputs.is_present }}
+    steps:
+      - name: Check for /test
+        id: test-command
+        run: |
+          comment_body=$(
+          cat <<'END_COMMENT'
+          ${{ github.event.review.body }}
+          END_COMMENT
+          )
+          if echo "$comment_body" | sed 's/\r$//' | grep -xq '^/test$'; then
+            echo "is_present=true" >> $GITHUB_OUTPUT
+          fi
+
+  kata-containers-ci-on-pr-review:
```

Comment:
Will "kata-containers-ci-on-pr-review" display on checkers table? If yes, could be make it shorter?

The jobs names are getting really long, even with big screen their names are truncated:

` Kata Containers CI / kata-containers-ci-on-push / run-metrics-tests / Kata Setup (...`

Perhaps even reduce this workflow name from `Kata Containers CI` to simply `CI`.

Reply:
Agreed. How about we make it look like `CI / on-review / run-metrics-tests /...` ? cc @fidencio 

Also gotta be mindful of what appears here: https://github.com/kata-containers/kata-containers/actions

Reply:
I'm fine with the suggestion!

Reply:
Done